justfile

```diff
@@ -73,8 +73,8 @@ deploy:
     @just up
 
 # Run pre-commit on all files
-lint:
-    pre-commit run --all-files
+lint hook="":
+    pre-commit run {{ hook }} --all-files
```

Comment:
Tweaked this so we can run `just lint isort`, the same way we do in the monorepo :)